test/unit/Grid.spec.ts

```diff
@@ -596,6 +626,7 @@ describe("test Grid", () => {
       const spy = sinon.spy();
       grid = new SampleGrid(container!, {
         autoResize: false,
+        resizeDebounce: 0,
```

Comment:
Why this option is added? `autoResize` is false so I think this isn't needed.
I'm asking this to check whether there isn't any mistake.